grafe-cli/src/generate/generate.ts

```diff
@@ -114,18 +205,17 @@ export async function generateMiddleWare(name: string, short: string, descriptio
     });
 
 
-    let _path = "/src/middlewares/" + short + "/";
+    let _path = path.join('src', 'middlewares', short);
 
-    await mkdirp(rootDir + _path);
+    await mkdirp(path.join(rootDir, _path));
 
-    _path += name + ".ts";
+    _path = path.join(_path, name + ".ts");
 
-    fs.writeFile(rootDir + _path, '', err => {
-        if (err) return console.log(err);
-        console.log("Created new Middleware " + _path);
-    });
+    const templateMiddleWarePath = path.join(__dirname, '..', '..', 'templates', 'starters', 'express', 'src', 'middlewares', 'pt', 'protected.ts');
```

Comment:
Da solltest deine Middleware Templates am besten in einen eigenen ordner machen. Vllt so `templates/middlewares/express.ts` oda so